.stylelintrc.yml

```diff
@@ -93,7 +93,9 @@ rules:
     selector-pseudo-element-no-unknown: true
     selector-type-case: lower
     selector-max-empty-lines: 0
-    selector-max-type: 0
+    selector-max-type:
+        - 0
+        - ignoreTypes: ["li"]
```

Comment:
Do you also want to exclude `img` and `span`? I'd guess that these are the most common element selectors that we're using. Not having to add a class everywhere might be a convenience-only feature though, so it's debatable.

Reply:
Hmm, we can add `i`, but `span` and `img` should be avoided to be honest.

Reply:
`img`: thinkable that you want to change an `img` with a `div` with a background image, that will break horribly.

`span`: should just be avoided except in very specific cases. In this case you should add the comment disabling the lint with a comment explaining it. (that is part of the purpose of this check)

Reply:
Oh, `body` and `html` should be allowed as well.
I wouldn't enforce the "class on the `body`" rule super strict, but just add it as comment in future PRs.